rocky/export_migrations/0070.tools.0029_delete_job.sql

```diff
@@ -0,0 +1,4 @@
+--
+-- Drop the job table
+--
+DROP TABLE "tools_job";
```

Comment:
I stopped adding these export sql files to git because @rdobeheer decided to create their own sql files that are incompatible with files that are automatically generated by django/sqlalchemy such as those in the export_migrations directory. Given that @rdobeheer can't use them and they are not useful for anyone else I think it is pointless continuing to keep maintaining these files in our git repository. Maybe we should also remove the existing ones?

Reply:
I have made a habit out of adding these upfront so we don't have to do this after the fact every time. If we officially drop that support, let's do it for every service at once in a separate PR perhaps? Not sure if that decision was finalized already.

Reply:
We still need these migrations to deploy to prod/staging... so I prefer to keep them... and keep them up-to-date